### PR TITLE
Improved the admin REST API

### PR DESCRIPTION
**Pull request by 1akshitha**
Improved the admin Rest API with 

1. Removing the unwanted cache control headers
2. Fixing the examples
3. Removing unwanted responses

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml [hunk 1]
```diff
@@ -1809,494 +1448,103 @@ paths:
...
-      - lang: Shell
-        source: 'curl -k -X DELETE -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
-          "https://127.0.0.1:9443/api/am/admin/v2/applications/0a043c2b-ee75-4ef3-9e1c-fc2610ccfa8b"'
-
-  /applications/{applicationId}/change-owner:
-    post:
-      tags:
-      - Application
-      summary: Change Application Owner
-      description: |
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml:1997]: Was this removal intentional?

Comment (1akshitha) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml:1997]: This is not a remove actually. Git diff that is displayed is not correct. These code segments are unchanged.

Comment (1akshitha) [reply components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml:1997]: https://github.com/wso2/carbon-apimgt/pull/9986/files#diff-833d66b2c304471172f7cf9a088df847a742e0f4832cb0ac10e3818165eae730R1616

**Review by malinthaprasan — APPROVED** [review]

**Comment by 1akshitha** [comment]
Fixes https://github.com/wso2/product-apim/issues/10294 and https://github.com/wso2/product-apim/issues/10296
